packages/isomorphic/src/utils/shared.ts

```diff
@@ -1,5 +1,7 @@
 import { concatBytes } from '@noble/hashes/utils'
 
+export const HEX_REGEX = /^[A-F0-9]*$/iu
```

Comment:
Do we want to change from `*` to `+` to prevent empty strings? @mvadari

Reply:
I purposefully kept support for empty strings - I don't see any reason why these functions shouldn't support empty strings, which would convert to an empty bytestring or empty ASCII string.

Reply:
Yes, but in what cases would an empty string be a legitimate input?

Reply:
One example: your app has a field that accepts memos and runs the result through `convertHexToString`